src/io/liteglue/SQLiteResponse.java

```diff
@@ -0,0 +1,20 @@
+package io.liteglue;
+
+public class SQLiteResponse {
+    private int result;
```

Comment:
well, previously the errors were also returned as long
shouldn't we keep it the same way?

Reply:
Well, [SQLite result codes](https://www.sqlite.org/c3ref/c_abort.html) are integers. They were returned as long because the return type had to match either the SQLite codes and the handle value. Also, the consumer was always casting the error value to a int value as you can see [here](https://github.com/OutSystems/Android-sqlite-connector/pull/1/files#diff-46984037bbcc46ac48874efa1e8eaa7e43c1f3bdc13f233e0c92441cfb5e4931L15)